include/crypto/hacl_hash.h

```diff
@@ -0,0 +1,435 @@
+/* 
+ * Copyright (c) 2016-2022 INRIA, CMU and Microsoft Corporation
+ * Copyright (c) 2022-2023 HACL* Contributors
+ * Copyright (c) 2023 Cryspen
+ */
+
+
+#ifndef __Hacl_Hash_H
```

Comment:
these headers are not public (in a sense - they should not be part of userspace headers)

better move them directly to `crypto` dir I think

Reply:
They are now in /crypto